docs/docsite/rst/getting_started/get_started_ansible.rst

```diff
@@ -1,65 +1,31 @@
 .. _get_started_ansible:
 
-*******************
-Start using Ansible
-*******************
+*****************************
+Start automating with Ansible
+*****************************
 
-Start automating with Ansible in a few easy steps.
+Get started with Ansible by creating an automation project, build an inventory, and creating a "Hello World" playbook.
```

Comment:
Get started with Ansible by creating an automation project. Build and inventory, and create a "Hello World" playbook.

Reply:
I'm not sure your suggestion improves grammar.